### DISCUSSION TITLE
HI I wonder if it’s possible to create the quikstart with sqlite , and later switch to mysql/Pg ?

### DISCUSSION THREAD

**Discussion by outboxltd** [strapi/strapi#7957] [Legacy Topics]
I just want to start with the basic guide and it includes sqlite?

**Comment by derrickmehaffy** [comment]
As answered on Slack, yes you can switch anytime:

![image](https://user-images.githubusercontent.com/8593673/93676805-6adf1d00-fa61-11ea-92d3-d1dcd7f7aea9.png)

https://strapi.io/documentation/v3.x/concepts/configurations.html#database
